docs/update_proxy.md

```diff
@@ -0,0 +1,38 @@
+# update_proxy
+
+Update the proxy configuration on the Rubrik cluster.
+```py
+def update_proxy(self, host, protocol, port, username=None, password=None, timeout=15)
+```
+
+## Arguments
+| Name        | Type | Description                                                                 | Choices |
+|-------------|------|-----------------------------------------------------------------------------|---------|
+| host  | str  | The IP address or FQDN of the proxy you wish to add. |         |
+| protocol  | str  | The protocol of the proxy you wish to add. |         |
+| port  | int  | The port of the proxy you wish to add. |         |
+
+## Keyword Arguments
+| Name        | Type | Description                                                                 | Choices | Default |
+|-------------|------|-----------------------------------------------------------------------------|---------|---------|
+| username | str  | The proxy username used for authentication.          |         |         |
```

Comment:
Similar to above, do we need `proxy` in the `username` and `password` descriptions.

Reply:
Set the `username` and `password` defaults to `None`